src/common/detectWMDE.c

```diff
@@ -275,6 +278,28 @@ static void getXFCE4(FFinstance* instance, FFWMDEResult* result)
     }
 }
 
+static void getLXQt(FFinstance* instance, FFWMDEResult* result)
+{
+    ffStrbufSetS(&result->deProcessName, "lxqt-session");
+    ffStrbufSetS(&result->dePrettyName, "LXQt");
+    ffParsePropFile("/usr/share/cmake/lxqt/lxqt-config.cmake", "set(LXQT_VERSION", &result->deVersion);
```

Comment:
As far as i know, cmake allows spaces, so we should too:
```c
"set ( LXQT_VERSION"
```
A space in the format string matches _any_ amount of spaces / tabs in the scanned string, even none.